x-pack/legacy/plugins/snapshot_restore/public/app/sections/home/recovery_list/recovery_table/shards_table.tsx

```diff
@@ -0,0 +1,252 @@
+        startTimeInMillis ? formatDate(startTimeInMillis) : <EuiLoadingSpinner size="m" />,
+    },
+    {
+      field: 'stopTimeInMillis',
+      name: i18n.translate('xpack.snapshotRestore.recoveryList.shardTable.endTimeColumnTitle', {
+        defaultMessage: 'End time',
+      }),
+      render: (stopTimeInMillis: SnapshotRecoveryShard['stopTimeInMillis']) =>
+        stopTimeInMillis ? formatDate(stopTimeInMillis) : <EuiLoadingSpinner size="m" />,
+    },
+    {
+      field: 'totalTimeInMillis',
+      name: i18n.translate('xpack.snapshotRestore.recoveryList.shardTable.durationColumnTitle', {
+        defaultMessage: 'Duration',
+      }),
+      render: (totalTimeInMillis: SnapshotRecoveryShard['totalTimeInMillis']) =>
+        totalTimeInMillis ? (
+          <FormattedMessage
+            id="xpack.snapshotRestore.recoveryList.shardTable.durationValue"
+            defaultMessage="{seconds} {seconds, plural, one {second} other {seconds}}"
+            values={{ seconds: Math.ceil(totalTimeInMillis / 1000) }}
+          />
+        ) : (
+          <EuiLoadingSpinner size="m" />
+        ),
+    },
+    {
+      field: 'repository',
+      name: i18n.translate('xpack.snapshotRestore.recoveryList.shardTable.repositoryColumnTitle', {
+        defaultMessage: 'Repository',
+      }),
+    },
+    {
+      field: 'snapshot',
+      name: i18n.translate('xpack.snapshotRestore.recoveryList.shardTable.snapshotColumnTitle', {
+        defaultMessage: 'Snapshot',
+      }),
+    },
+    {
+      field: 'version',
+      name: i18n.translate('xpack.snapshotRestore.recoveryList.shardTable.versionColumnTitle', {
+        defaultMessage: 'Version',
+      }),
+    },
+    {
+      field: 'targetHost',
+      name: i18n.translate('xpack.snapshotRestore.recoveryList.shardTable.targetHostColumnTitle', {
+        defaultMessage: 'Target host',
+      }),
+    },
+    {
+      field: 'targetNode',
+      name: i18n.translate('xpack.snapshotRestore.recoveryList.shardTable.targetNodeColumnTitle', {
+        defaultMessage: 'Target node',
+      }),
+    },
+    {
+      field: 'bytesTotal',
+      name: i18n.translate('xpack.snapshotRestore.recoveryList.shardTable.bytesColumnTitle', {
+        defaultMessage: 'Bytes',
+      }),
+      render: (
+        bytesTotal: SnapshotRecoveryShard['bytesTotal'],
+        { bytesRecovered, bytesPercent }: SnapshotRecoveryShard
+      ) => {
+        return (
+          <EuiToolTip
+            position="top"
+            content={i18n.translate(
+              'xpack.snapshotRestore.recoveryList.shardTable.bytesTooltipLabel',
+              {
+                defaultMessage: 'Recovered: {recovered}, Total: {total}',
+                values: {
+                  recovered: bytesRecovered,
+                  total: bytesTotal,
+                },
+              }
+            )}
+          >
+            <EuiText size="xs" textAlign="center" style={{ width: '100%' }}>
```

Comment:
Can this inline style be added to a `.scss` file?

Reply:
this was actually no longer needed 😄